provision/ec2.go

```diff
@@ -165,39 +166,70 @@ func (p *EC2Provisioner) CreateInstance(opts EC2CreateInstanceOptions) (*cfg.Rem
 		return nil, err
 	}
 
+	// Check response validity
+	if runResp.Instances == nil || len(runResp.Instances) == 0 {
+		return nil, errors.New("Unable to start instances: " + runResp.String())
+	}
+
 	// Loop until intance is running
-	fmt.Fprint(p.out, "Checking status of requested instance...")
-	attempts := 0
-	var instanceStatus *ec2.DescribeInstancesOutput
+	fmt.Fprintln(p.out, "Checking status of requested instance...")
+	var (
+		attempts = 0
+		instance ec2.Instance
+	)
 	for {
+		// Wait briefly between checks
+		time.Sleep(3 * time.Second)
 		attempts++
```

Comment:
This seems to be a unused variable. Perhaps you wanted to increase the check time period?

Reply:
Right, I think this was supposed to be a cap on the number of attempts. I might remove it entirely.